### ISSUE TITLE
#391 seems to break building of buildifier

### ISSUE THREAD

**Issue by madeddie** [bazelbuild/buildtools#392]
- go version go1.11.1 darwin/amd64
- commit 89798e3ab8296f81e0d5468b2c0f8b27a18244bd

```
$ rm -rf ~/devel/_go/{src,bin}/
$ go get -v github.com/bazelbuild/buildtools/buildifier
github.com/bazelbuild/buildtools (download)
package github.com/bazelbuild/buildtools/warn: found packages bzlenv (bzlenv.go) and warn (warn.go) in /Users/edwin/devel/_go/src/github.com/bazelbuild/buildtools/warn
```

When I check out the previous commit, it finishes without messages.
```
$ git checkout f2cf333bdaea52ca7287f1c4fe6f7e3182356bf7
Note: checking out 'f2cf333bdaea52ca7287f1c4fe6f7e3182356bf7'.

You are in 'detached HEAD' state. You can look around, make experimental
<SNIP>
HEAD is now at f2cf333 Fix the anchors (#390)
$ go get -v github.com/bazelbuild/buildtools/buildifier
$ ls ~/devel/_go/bin/
buildifier
```

**Comment by pmbethe09** [comment]
is this still true?

**Comment by vladmos** [comment]
It's been fixed with #393
